client/state/countries/selectors.js

```diff
@@ -0,0 +1,7 @@
+export function getCountries( state, type ) {
+	return state.countries.items[ type ];
+}
+
+export function isFetching( state, type ) {
```

Comment:
Right, I was more thinking of the usefulness of param docs for these two selectors